paddle/fluid/inference/tests/api/analyzer_vit_ocr_tester.cc

```diff
@@ -108,7 +108,7 @@ TEST(Analyzer_vit_ocr, fuse_status) {
       static_cast<AnalysisPredictor *>(predictor.get()), &num_ops);
 
   CHECK_EQ(fuse_status.at("fc_mkldnn_pass"), 33);
-  CHECK_EQ(fuse_status.at("conv_activation_mkldnn_fuse"), 2);
+  CHECK_EQ(fuse_status.at("conv2d_gelu_mkldnn_fuse_pass"), 2);
```

Comment:
Hi I thought `conv2d_gelu_mkldnn_fuse_pass` has been deleted, why it is still here ?

Reply:
This part needs further analysis and refactoring, because current approach can be misleading and it also affects other passes with activations.

`fuse_status.at()` is checking how many ops should be fused in optimized version of model. This information is added via `AddStatis()` method. Unfortunately, approach with bulk fusion of conv+activation returns number of fusion only for last pair of fused `conv+act`.  For example only how many `conv+mish` were fused, instead of cumulated sum for all `conv+activations`. That's why for now I'm saving number of each conv+activation individually.